FWCore/Framework/src/EventProcessor.cc

```diff
@@ -1351,11 +1350,11 @@ namespace edm {
     unsigned int streamIndex = 0;
     for (; streamIndex < preallocations_.numberOfStreams() - 1; ++streamIndex) {
       tbb::task::enqueue(*edm::make_functor_task(tbb::task::allocate_root(), [this, streamIndex, h = iHolder]() {
-        handleNextEventForStreamAsync(std::move(h), streamIndex);
+        handleNextEventForStreamAsync(h, streamIndex);
```

Comment:
What is the motivation behind removing std::move here and below? I can see it will still work, but just wondering why? The rest  of the PR looks nice.

Reply:
That was changed by `scram b code-checks`. It said the `std::move` was being applied to a `const &` so the move was not doing anything. One thing to note, all captured variables are considered `const` unless the lambda is marked as `mutable`.